plugins/woocommerce-blocks/assets/js/blocks/mini-cart/style.scss

```diff
@@ -33,7 +33,6 @@
 	padding: em($gap-small) em($gap-smaller);
 
 	&:hover:not([disabled]) {
-		opacity: 0.6;
```

Comment:
There is no way to maintain any opacity value and guarantee that the hover style color will pass the WCAG color contrast rule. 
Since the customer-account icon doesn't change opacity when hovered over and these two icons are usually displayed together, I believe it makes sense to keep their hover styles consistent by removing the opacity from the mini-cart button style.

![image](https://github.com/user-attachments/assets/d28ddf6f-bb9b-46fd-bf1e-fbde8a91e974)
![image](https://github.com/user-attachments/assets/1fb396f0-7bde-4ce7-9234-1d88833cdf89)